### ISSUE TITLE
Unblock other ports

### ISSUE THREAD

**Issue by GoogleCodeExporter** [soul4soul/peerblock#255]
```
Please add the capability to unblock additional ports (eg SMTP)
```

Original issue reported on code.google.com by `glen...@tgt.id.au` on 17 Jan 2010 at 4:05
- Merged into: #12

**Comment by GoogleCodeExporter** [comment]
```
This feature request is actually being tracked by Issue #12.  Work on it is 
currently
in-progress, and should be included in the next Beta Release!
```

Original comment by `peerbloc...@gmail.com` on 17 Jan 2010 at 4:53
- Changed state: **Duplicate**
